Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:latest
+
+WORKDIR /home/app/wca-documents
+
+# Install the required dependencies
+COPY ./bin/install_dependencies.sh .
+RUN ./install_dependencies.sh
+RUN rm ./install_dependencies.sh
+
+# All of this makes it so we can build PDFs without having them owned by root
+ARG USER_ID
+ARG GROUP_ID
+RUN groupadd --non-unique --gid $GROUP_ID user
+RUN useradd --non-unique --uid $USER_ID --gid $GROUP_ID user
+USER user
```

Comment:
Out of curiosity: The above `WORKDIR /home/app` path kinda implies that there is already a user called `app` pre-defined by the Ubuntu image. Does that one have the necessary privileges?

Reply:
This is the output I get when doing ls -l /home/app:

`drwxr-xr-x 11 ubuntu ubuntu 4096 May 28 10:06 wca-documents`

Reply:
That does not fully answer my question. Does the user `app` actually exist or am I misreading some file/folder configuration?
If the user exists, please try out whether it has the necessary privileges for running wasyprint. I cannot figure this out by staring at `chmod`esque permission return values.

Reply:
Oh, my bad, I misunderstood. The app user doesn't exist in the ubuntu docker image, no. This directory can simply be used for your app's files.